integration/pubsub_test.go

```diff
@@ -227,6 +254,93 @@ func (suite *serviceBusSuite) duplicateDetectionTest(pub *publisher.Publisher, l
 	)
 }
 
+func (suite *serviceBusSuite) publishAndConcurrentReceiveMessage(testConfig publishReceiveTest, numOfEvents int) {
+	ctx := context.Background()
+	if testConfig.listenerOptions == nil {
+		testConfig.listenerOptions = []listener.Option{}
+	}
+	if testConfig.publisherOptions == nil {
+		testConfig.publisherOptions = []publisher.Option{}
+	}
+
+	events := []testEvent{}
+	for i := 0; i < numOfEvents; i++ {
+		j := i + 1
+		event := testEvent{
```

Comment:
this could just be anonymous empty struct we're not actually using the data here.